jjb/mkck/mkck-trigger.yaml

```diff
@@ -514,9 +554,18 @@
             TARGET_IP=$addr
             TARGET_NAME=$name
             EOF
-
         - inject:
             properties-file: $TARGET_FILE
+        - shell: |
+            . v/bin/activate
+            . sesci/common/util
+            for i in artifacts*.yaml ; do
+                # now jq black magic
```

Comment:
hm. can't we just use code from `common/sesci` or merge the `artifacts2zypper.py` script from https://gitlab.suse.de/ceph/storage-ci/-/merge_requests/110/diffs?commit_id=b54db153166839faaaee40d8f93de39c5a667eb7#diff-content-b3d020c337ec05b679cafccd1f7a8ed2cb755cbe and use that?

Reply:
I don't see any benefit in artifacts2zypper.py 38 lines of pythonism, in comparison to one liner jq command. I mean it does not worth to create a separate python script which can be done with single command. As for common/sesci it is a separate repo too.
Btw, as for artifacts2zypper.py, maybe there is a sense to add format key for artifacts.py to jenerate zypper config file or zypper command.

Reply:
`artifacts2zypper.py` does some error checking. And one-liners are imho not always better. This one liner is just difficult to get correct...
Anyway, I see you are in favor of using this approach